### PR TITLE
[c3] update readme with examples

### PR DESCRIPTION
**Pull request by admah**
Fixes # [insert GH or internal issue number(s)].

**What this PR solves / how to test:**

This PR updates the C3 README to include examples and additional arguments that users can use to create new apps via CLI.

**Associated docs issue(s)/PR(s):**

- [insert associated docs issue(s)/PR(s)]

**Author has included the following, where applicable:**

- [ ] Tests
- [ ] Changeset ([Changeset guidelines](https://github.com/cloudflare/workers-sdk/blob/main/CONTRIBUTING.md#changesets))

**Reviewer is to perform the following, as applicable:**

- Checked for inclusion of relevant tests
- Checked for inclusion of a relevant changeset
- Checked for creation of associated docs updates
- Manually pulled down the changes and spot-tested

### CODE REVIEW

File: packages/create-cloudflare/README.md [hunk 1]
```diff
@@ -10,7 +16,44 @@ pnpm create cloudflare@latest
...
 ```
 
+### Setup via CLI Arguments
+
+#### New Websites or Web applications via Frameworks
+
+To create a new website or web framework without interaction, run:
+
+```bash
+npm create cloudflare@latest -- --type webFramework -- --framework <frameworkName>
```
Comment (petebacondarwin) [inline packages/create-cloudflare/README.md:26]: There are two `--` markers here. Is that correct?

File: packages/create-cloudflare/README.md [hunk 2]
```diff
@@ -10,7 +16,44 @@ pnpm create cloudflare@latest
...
+# or
+pnpm create cloudflare@latest
```
Comment (petebacondarwin) [inline packages/create-cloudflare/README.md:28]: Perhaps the following to indicate that you still need to add the parameters.
```suggestion
pnpm create cloudflare@latest ...
```

File: packages/create-cloudflare/README.md [hunk 3]
```diff
@@ -10,7 +16,44 @@ pnpm create cloudflare@latest
...
+
+#### Additional arguments
+
+| `--deploy` | deploy your application automatically, bypassing the interactive prompt |
+| `--no-deploy` | create and scaffold a new application and bypass deployment prompt |
+
 ### Community
 
-- Join us [on Discord](https://discord.gg/cloudflaredev)
+- Join us [on Discord](https://discord.cloudflare.com)
```
Comment (petebacondarwin) [inline packages/create-cloudflare/README.md:58]: :+1:

File: packages/create-cloudflare/README.md (hunk 1 above)
Comment (admah) [reply packages/create-cloudflare/README.md]: Good catch. I removed those.

File: packages/create-cloudflare/README.md [hunk 4]
```diff
@@ -10,7 +16,44 @@ pnpm create cloudflare@latest
...
+```
+
+Currently supported framework options: `angular`, `astro`, `docusaurus`, `gatsby`, `hono`, `next`, `nuxt`, `qwik`, `react`, `remix`, `solid`, `svelte`, `vue`.
+
+#### New Workers via Templates
+
+To create a new Javascript "Hello World" worker, run:
+
+```bash
+npm create cloudflare@latest hello-world -- --type simple --ts false
```
[the remark this answers is not in the document]
Comment (admah) [reply packages/create-cloudflare/README.md]: Added that flag.

File: packages/create-cloudflare/README.md [hunk 5]
```diff
@@ -37,7 +37,7 @@ Currently supported framework options: `angular`, `astro`, `docusaurus`, `gatsby
 To create a new Javascript "Hello World" worker, run:
 
 ```bash
-npm create cloudflare@latest hello-world -- --type simple --ts false
+npm create cloudflare@latest hello-world -- --type simple --no-ts
```
Comment (Skye-31) [inline packages/create-cloudflare/README.md:40]: This still has the extra `--` 🙂

**Review by jculvey — APPROVED** [review]

File: packages/create-cloudflare/README.md (hunk 5 above)
Comment (admah) [reply packages/create-cloudflare/README.md:40]: @Skye-31 `npm` actually requires the first set so it'll accept those args. The difference before was I had two sets of `--` like this: 

`npm create cloudflare@latest -- --type webFramework -- --framework <frameworkName>`

Comment (Skye-31) [reply packages/create-cloudflare/README.md:40]: Ah I misread the initial one then 😅

**Pull request merged**